src/components/Form.js

```diff
@@ -163,9 +191,115 @@ export default function Form({ handleActivity, preloadedValues, title }) {
+            />
+            <input
+              id="selectPhoto"
+              name="selectPhoto"
+              type="file"
+              onChange={uploadImage}
+              hidden
+            />
+          </label>
 
-      <label htmlFor="date">
+          {preloadedPicture ? (
+            <PositionedSection>
+              <div>
+                <StyledImage
+                  width="60"
+                  height="60"
+                  alt={`preview ${preloadedPicture}`}
+                  src={preloadedPicture}
+                />
+              </div>
+              <DeletePictureButton onClick={e => onDeletePreloadedPicture(e)}>
+                <img
+                  src={deletePictureIcon}
+                  alt="delete"
+                  width="20"
+                  height="20"
+                />
+              </DeletePictureButton>
+            </PositionedSection>
+          ) : (
+            <div>
+              {photo ? (
+                <PositionedSection>
+                  <StyledImage
+                    width="60"
+                    height="60"
+                    alt={`preview ${photo}`}
+                    src={photo}
+                  />
+                  {photo && (
+                    <DeletePictureButton onClick={e => onDeletePicture(e)}>
+                      <img
+                        src={deletePictureIcon}
+                        alt="delete"
+                        width="20"
+                        height="20"
+                      />
+                    </DeletePictureButton>
+                  )}
+                </PositionedSection>
+              ) : (
+                <StyledPreviewText>preview</StyledPreviewText>
+              )}
+            </div>
+          )}
+        </StyledPictureUpload>
+      ) : (
+        <StyledPictureUpload>
+          <label htmlFor="selectPhoto">
+            <img
+              width="50"
+              height="50"
+              alt="selectPhoto"
+              src={addPictureIcon}
+            />
+            <input
+              id="selectPhoto"
+              name="selectPhoto"
+              type="file"
+              onChange={uploadImage}
+              hidden
+            />
+          </label>
+          {photo ? (
+            <PositionedSection>
+              <StyledImage
+                width="60"
+                height="60"
+                alt={`preview ${photo}`}
+                src={photo}
+              />
+              {photo && (
+                <DeletePictureButton onClick={e => onDeletePicture(e)}>
+                  <img
+                    src={deletePictureIcon}
+                    alt="delete"
+                    width="20"
+                    height="20"
+                  />
+                </DeletePictureButton>
+              )}
+            </PositionedSection>
+          ) : (
+            <StyledPreviewText>preview</StyledPreviewText>
+          )}
+        </StyledPictureUpload>
+      )}
+
```

Comment:
this was horrible to read :) but I think i know what you did and for me it looks good, but it is a lot of code... :D